### PR TITLE
Added a new Mod to the emulator 

### LINKED ISSUE #62: Create map
As a driver, I want to change the environment for my car, to a more fitting location like a road to test its features. 

Acceptance criteria:
- [ ] Create a new map on blender
- [ ] Export the object/map from blender and import the file to Godot Engine 
- [ ] Customize the map/objects with collision features 
- [ ] Create obstacles for the car to avoid 
- [ ] Create script so the map can execute on SMCE Godot 

Optional Acceptance criteria: 
- [ ] The obstacles will take the shape of a car 
- [ ] The obstacles/cars are moving

### PR DESCRIPTION
**Pull request by AlbinKarlsson**
## Created a map 

Issue: Closes #62 

- Created a road on blender 
- Created a godot file 
   - Imported the road to the godot project
- Used godot engine 
- Created a world to drive on 
   - Added RigidBodies and collision rules 
- Created a script to initialize the new mod
- Added a folder called mods in the repository 
   - Added all the files that I used for my world (.pck and .tscn) 

Before this branch the project used a map provided by the class. This mod and customized map is more fitting for the project. 

### :ballot_box_with_check: Definition of Done checklist
- [x] Meaningful title and valuable description in pull request
- [x] Code change tested manually if not covered by automated tests
- [x] Pull request linked to one or more issues/stories
- [x] Acceptance criteria of the linked stories satisfied
- [x] Code change comprehended by those who approved it
- [x] State of target branch maintained or improved after the code change

### CODE REVIEW

**Review by Valkama1 — COMMENTED** [review]
I have one complaint. Maybe the map file name should be called something other than "test.pck"?

File: mods/ReadMe.txt [hunk 1]
```diff
@@ -0,0 +1,63 @@
+||COPYRIGHT|| - ||Patent pending|| - ||CREATOR: Albin Karlsson|| - ||UNIVERSITY OF GOTHENBURG||
```
Comment (Lindet94) [inline mods/ReadMe.txt:1]: Good job on the readme describing how to access the new map. I was able to follow it and it worked.

**Review by Lindet94 — APPROVED** [review]
Looks good Albin!

Good job on the Readme. I've read it and I was able to run the new environment without issues by following the steps.

I saw that there is a mod called test.pck. This should be included? I think it's fine to have it in the master so I will still approve the pull request.

**Review by Lindet94 — CHANGES_REQUESTED** [review]
This was a mistake. I think it should be approved , but I'm not sure how to reverse this... :)

**Review by Lindet94 — APPROVED** [review]

**Pull request merged**